src/terra-dev-site/doc/reference.1/components/ApplicationBase.doc.mdx

```diff
@@ -0,0 +1,43 @@
+import PropsTable from 'terra-application/src/application-base/ApplicationBase?dev-site-props-table';
+
+# ApplicationBase
+
+The ApplicationBase component is the entrypoint into the Terra application framework.
+
+> Note: ApplicationBase satisfies the base-level requirements for Terra component usage. Rendering a separate Base component from `terra-base` is not necessary.
+
+## Usage
+
+```jsx
+import ApplicationBase from 'terra-application/lib/application-base';
+```
+
+## Props
+
+<PropsTable />
```

Comment:
![](https://media.giphy.com/media/9DnzBDe9bF06OICDfQ/giphy.gif)

Reply:
It works like a charm